x/dispensation/types/types.go

```diff
@@ -1,15 +1,12 @@
 package types
 
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
 // ----------------------------------------------------------------------------
 // Client Types
 
-type DistributionRecordsResponse struct {
-	DistributionRecords
-	Height int64 `json:"height"`
-}
-
-func NewDistributionRecordsResponse(distributionRecords DistributionRecords, height int64) DistributionRecordsResponse {
-	return DistributionRecordsResponse{DistributionRecords: distributionRecords, Height: height}
+func NewDistributionRecordsResponse(distributionRecords DistributionRecords, height sdk.Int) DistributionRecordsResponse {
+	return DistributionRecordsResponse{DistributionRecords: &distributionRecords, Height: height}
 }
 
 type DistributionsResponse struct {
```

Comment:
Curious as to why the DistributionsResponse , not defined through proto

Reply:
I've defined this in proto now.